src/middleware/verifyInviteToken.js

```diff
@@ -0,0 +1,39 @@
+const jwt = require('jsonwebtoken')
+const config = require('../config')
+
+
+const verify = (req, res, next) => {
+	const authHeader = req.headers.authorization
+
+	if (!authHeader){
+		res.status(401).json({
+			status: "fail",
```

Comment:
@itzgeebee, I think the `AppError` function already does the  setting of  `status` for us 

we can have uniformity for this by passing a `new AppError` function as a parameter to the `next` function 

also after the change, kindly add a `return` before it. it seems we are getting an ` Cannot set headers after they are sent to the client` when there's an error because the subsequent  lines of code still run